.github/workflows/ci.yml

```diff
@@ -1,246 +0,0 @@
-name: CI
```

Comment:
Why was this deleted? Are there restrictions on GitHub that would prevent us from running both workflows?